modules/ting_search/ting_search.module

```diff
@@ -4,6 +4,8 @@
  * Enables search integration with Ting.
  */
 
+use Ting\Search\TingSearchResultInterface;
```

Comment:
Is there a reason why this interface is imported? As far as I can tell it is not actually used in this class. `TingClientSearchResult` is though.

Reply:
whops :) - will fix

Reply:
TingClientSearchResult is in the default namespace - so that why there is no import. But you have a point regarding TingSearchResultInterface...